static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -49,6 +49,13 @@ function EditAccessSelector({dashboard, onChangeEditAccess}: EditAccessSelectorP
       newSelectedValues.includes(teamId)
     );
 
+    // Enables/Disables the 'Save Changes' button
+    if (isEqual(newSelectedValues, getSelectedOptions())) {
+      setHasUnsavedChanges(false);
+    } else {
+      setHasUnsavedChanges(true);
+    }
+
```

Comment:
Can we just pass `isEqual(newSelectedValues, getSelectedOptions())` into `disabled` on line 207? so `disabed={!isCurrentUserDashboardOwner || isEqual(newSelectedValues, getSelectedOptions())}`, or you can just assign it to a variable instead of storing it in state like this